README_en.md

```diff
@@ -21,467 +21,97 @@ PaddleClas is a toolset for image classification tasks prepared for the industry
 
 - [more](./docs/en/update_history_en.md)
 
-
 ## Features
 
-- Rich model zoo. Based on the ImageNet-1k classification dataset, PaddleClas provides 29 series of classification network structures and training configurations, 134 models' pretrained weights and their evaluation metrics.
-
-- SSLD Knowledge Distillation. Based on this SSLD distillation strategy, the top-1 acc of the distilled model is generally increased by more than 3%.
+- A practical image recognition system: Integrate modules for detection, feature learning and retrieval, widely applicable to all types of image recognition tasks.
```

Comment:
A practical image recognition system consist of detection, feature learning and retrieval modules, ...